### PR TITLE
🚀 monitor

### PR DESCRIPTION
**Pull request by thibaultleouay**
An example how to fetch the data  for the different pages 🚀

### CODE REVIEW

File: apps/web/src/middleware.ts [hunk 1]
```diff
@@ -37,14 +42,19 @@ export default authMiddleware({
...
         .execute();
+
       if (result.length) {
         const orgSelection = new URL(
           `/app/${result[0].users_to_workspaces.workspaceId}`,
           req.url,
         );
         return NextResponse.redirect(orgSelection);
+      } else {
+        // return NextResponse.redirect(new URL("/app/onboarding", req.url));
```
Comment (thibaultleouay) [inline apps/web/src/middleware.ts:55]: @mxkaske that's where we should probably do something when the webhook is not received yet. 

redirect to a small onboarding page?

File: apps/web/src/app/app/(dashboard)/[workspaceId]/monitor/page.tsx [hunk 2]
```diff
@@ -2,10 +2,18 @@ import * as React from "react";
...
 
-export default async function MonitorPage() {
-  await wait(1000);
+export default async function MonitorPage({
+  params,
+}: {
+  params: { workspaceId: string };
+}) {
+  const monitors = await api.monitor.getMonitorsByWorkspace.query({
+    workspaceId: Number(params.workspaceId),
```
Comment (thibaultleouay) [inline apps/web/src/app/app/(dashboard)/[workspaceId]/monitor/page.tsx:13]: Here we fetch the monitors 🚀
Some logic for all the components

File: apps/web/src/middleware.ts (hunk 1 above)
Comment (mxkaske) [reply apps/web/src/middleware.ts:55]: Let's see what we can do. I don't mind having a Loading screen with a small description what is happening. Being open about the status.

**Review by mxkaske — APPROVED** [review]
Lgtm!

**Pull request merged**